test_files/abstract/abstract.js

```diff
@@ -1,5 +1,6 @@
 /**
  * @fileoverview added by tsickle
+ * Generated from: test_files/abstract/abstract.ts
```

Comment:
It's a bit odd that we add an implementation detail comment like this into a doc comment (`/** */`), but we need to add the doc comment anyway due to JSCompiler only processing `@suppress` directives in JSDoc. Given that, I think this is the right place, there's no point in adding another `// ` comment.